pkg/controllers/disruption/drift.go

```diff
@@ -76,7 +75,6 @@ func (d *Drift) ComputeCommand(ctx context.Context, candidates ...*Candidate) (C
 	if err != nil {
 		return Command{}, err
 	}
-	deprovisioningEligibleMachinesGauge.WithLabelValues(d.Type()).Set(float64(len(candidates)))
```

Comment:
Is this coupled to this work? Is there a way to separate into another PR?

Reply:
Nope. I have a list of alpha changes I've made here that i'll make in a separate PR prior to merging this one.